### PR TITLE
[13.0][MIG] mass_mailing_event_registration_exclude

### PR DESCRIPTION
**Pull request by fuentes73**
Standard migration to 13.0

### CODE REVIEW

**Review by Abranes — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@chienandalu TT26226

File: mass_mailing_event_registration_exclude/data/event_state_data.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
```
Comment (victoralmau) [inline mass_mailing_event_registration_exclude/data/event_state_data.xml]:
```suggestion
<odoo noupdate="1">
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Good job!

File: mass_mailing_event_registration_exclude/data/event_state_data.xml (hunk 1 above)
Comment (fuentes73) [reply mass_mailing_event_registration_exclude/data/event_state_data.xml]: @victoralmau done. Good catch!

**Review by victoralmau — APPROVED** [review]

File: mass_mailing_event_registration_exclude/data/event_state_data.xml (hunk 1 above)
Comment (chienandalu) [reply mass_mailing_event_registration_exclude/data/event_state_data.xml]: Sorry, why `noupdate`? The attendee states won't change.

**Pull request merged**